docker/docker_transport_test.go

```diff
@@ -181,6 +255,13 @@ func TestReferenceNewImageDestination(t *testing.T) {
 		&types.SystemContext{RegistriesDirPath: "/this/does/not/exist", DockerPerHostCertDirPath: "/this/does/not/exist"})
 	require.NoError(t, err)
 	defer dest.Close()
+
+	ref, err = ParseReference("//quay.io/libpod/busybox" + UnknownDigestSuffix)
+	require.NoError(t, err)
+	dest2, err := ref.NewImageDestination(context.Background(),
+		&types.SystemContext{RegistriesDirPath: "/this/does/not/exist", DockerPerHostCertDirPath: "/this/does/not/exist"})
+	require.NoError(t, err)
+	defer dest2.Close()
 }
 
 func TestReferenceTagOrDigest(t *testing.T) {
```

Comment:
Also test the `isUnknownDigest` code path here

Reply:
Fixed